### DISCUSSION TITLE
Guidance on re-using a shared SetupSuite across multiple packages

### DISCUSSION THREAD

**Discussion by Oscmage** [stretchr/testify#1887] [Q&A]
Hi, thanks for this lovely package. 

I wonder if it is possible to use a suite across multiple packages?

I have an integration test use case where I'd like to start a container that is expensive in terms of time. I'd like to re-use this container at something like a SetupSuite stage and then tear it down when the full integration suite is complete across all package. I have no problem achieving this when everything is within a shared package. The twist of the problem is that I'd like to re-use this container across multiple packages.

Is my best option to simply put all of my integration tests in one huge package or is there a more clever way to achieve this that doesn't feel like a "hack"?

Thanks!

**Comment by dolmen** [comment]
Just put your Suite in its own package imported where necessary.
